Model.cpp

```diff
@@ -75,162 +75,127 @@ void Model::editParameter(const std::string& className, const std::string& metho
 
 void Model::deleteParameter(const std::string& className, const std::string& methodName, const std::string& ParameterName)
 {
-}
-
-void Model::addRelationship(const std::string & parent, const std::string & child, RelationshipType type)
-{
-	auto p = find(mClasses.begin(), mClasses.end(), parent);
-	auto c = find(mClasses.begin(), mClasses.end(), child);
-	mRelationships.push_back(Relationship((*p), (*c), type));
+}
+
+void Model::addRelationship(const std::string & parent, const std::string & child, RelationshipType type)
+{
+	auto p = find(mClasses.begin(), mClasses.end(), parent);

```

Comment:
p may not exist. c may not exist. let the model dictate the type of error that should be presented to the user, but don't have the model print it or display it